src/main/java/gov/nasa/pds/api/engineering/controllers/MyProductsApiBareController.java

```diff
@@ -93,6 +94,36 @@ public MyProductsApiBareController(ObjectMapper objectMapper, HttpServletRequest
 	        
     }
     
+    protected Products getLIDVIDs(List<String> lidvids, List<String> fields, HashSet<String> uniqueFields, boolean onlySummary) throws IOException
+    {
+    	String[] aFields = new String[fields == null ? 0 : fields.size()];
+    	if (fields != null)
+    	{
+    		for (int i = 0 ;  i < fields.size(); i++) aFields[i] = ElasticSearchUtil.jsonPropertyToElasticProperty(fields.get(i));
+    	}
+
+    	BoolQueryBuilder find_lidvids = QueryBuilders.boolQuery();
+    	Products results = new Products();
+    	SearchRequest request = new SearchRequest(this.esRegistryConnection.getRegistryIndex())
+    			.source(new SearchSourceBuilder().query(find_lidvids)
+    					.fetchSource(fields == null ? this.DEFAULT_ALL_FIELDS : aFields, this.DEFAULT_BLOB));
+
+    	for (String lidvid : lidvids) find_lidvids.should (QueryBuilders.termQuery ("lidvid", lidvid));
+    	for (SearchHit hit : this.esRegistryConnection.getRestHighLevelClient().search(request, RequestOptions.DEFAULT).getHits())
+    	{
+    		Map<String, Object> response = ElasticSearchUtil.elasticHashMapToJsonHashMap(hit.getSourceAsMap());
```

Comment:
I replace this function with one in the latest master, see https://github.com/NASA-PDS/registry-api-service/blob/ee3af9c844f573a256926f02147bcd774662d472/src/main/java/gov/nasa/pds/api/engineering/elasticsearch/business/ProductBusinessObject.java#L125

Reply:
Can you run:
`git rebase master`

To remove the conflicts

Reply:
@tloubrieu-jpl @jordanpadams 

The function getFilteredProperties() is not the same as these lines of code outlines. The function requires all data being loaded then filtering takes place in the Java code while the outlined code has ES do the work as #13 requires. Please resolve conflict by changing the #13 requirements of being efficient or not using the function getFilteredProperties().